wiki.py

```diff
@@ -361,7 +361,7 @@ def _get_projects_for_strategy(self, strategy):
 
         """
 
-        for number, name in self._projects.items():
+        for number in self._projects:
```

Comment:
might be more clear to do `for number in self._projects.keys():` this is what we use elsewhere